### DISCUSSION TITLE
Spacing Issue in P-Editor and also Disappearing few lines

### DISCUSSION THREAD

**Discussion by Samratgoud13** [primefaces/community#367] [PrimeNG] [answered]
In P-Editor Prime NG editor content format is not working. 
There is a spacing issue in the below Output of sample data.

**Below is the sample data which I sent:**

1.Contrary to popular belief, Lorem Ipsum is not simply random text. 
2.It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. 
3.Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, 
and going through the cites of the word in classical literature, discovered the undoubtable source. 
4.Lorem Ipsum comes from sections 1.10.32 and 1.10.33 of "de Finibus Bonorum et Malorum" (The Extremes of Good and Evil) by Cicero, written in 45 BC. 
5.This book is a treatise on the theory of ethics, very popular during the Renaissance. The first line of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes from a line in section 1.10.32.

**But I am receiving the below output:**

1.Contrarytopopularbelief,LoremIpsumisnotsimplyrandomtext. 
2.IthasrootsinapieceofclassicalLatinliteraturefrom45BC,makingitover2000yearsold. 
3.RichardMcClintock,aLatinprofessoratHampden-SydneyCollegeinVirginia,lookeduponeofthemoreobscureLatinwords,consectetur,fromaLoremIpsumpassage, 
andgoingthroughthecitesofthewordinclassicalliterature,discoveredtheundoubtablesource. 
4.LoremIpsumcomesfromsections1.10.32and1.10.33of"deFinibusBonorumetMalorum"(TheExtremesofGoodandEvil)byCicero,writtenin45BC. 
5.Thisbookisatreatiseonthetheoryofethics,verypopularduringtheRenaissance.ThefirstlineofLoremIpsum,"Loremipsumdolorsitamet..",comesfromalineinsection1.10.32.

Could anyone please help me out with this issue???

**Comment by cetincakiroglu** [comment]
Hi,

The issue is not clear, could you please share a stackblitz example so we can identify the issue clearly?

**Comment by Samratgoud13** [comment]
I am trying to give this sample data.
![image](https://github.com/primefaces/community/assets/149620209/6939ca6b-aba7-4f79-9dfb-a7eaf324e496)

But I am getting this output when I paste in P-Editor.
![image](https://github.com/primefaces/community/assets/149620209/f5f57c73-db11-41c9-9b9d-3eeea4d4188e)

**Comment by Samratgoud13** [comment]
[Allignment-change-Issue-Email-.zip](https://github.com/primefaces/community/files/13285560/Allignment-change-Issue-Email-.zip)

To get clear idea.

**Reply by cetincakiroglu** [reply]
Do not share a zip, please share a stackblitz url so we can identify the issue clearly

**Comment by Samratgoud13** [comment]
[This is for Stackblitz you find the issue by pasting the Sample Data] 
I am attaching the sample data in word document. I wanted to get the same format which is in the word document. 

[Format Issue Sample Data.docx](https://github.com/primefaces/community/files/13299307/Format.Issue.Sample.Data.docx)



(https://stackblitz.com/edit/primeng-editor-demo-5rrksj?file=src%2Fapp%2Fapp.module.ts,src%2Fapp%2Fapp.component.ts)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We're using quill as prime editor and we'll deprecate the quill dependency in the future. It would be better for you to use more modern and updated editors. We're just wrapping the editor by p-editor and provide some extra styling. Logic is not controlled by PrimeNG actually.

**Reply by ThoSap** [reply]
As you can see Quill is again under active development since September 2023.
The new v2.0 version, which will be released soon no longer has the mutation events which will be removed from Chrome/Chromium on July 2024 (Chromium 127).
https://github.com/quilljs/quill/issues/3806#issuecomment-1933250524
https://github.com/quilljs/quill/releases

I do not think the community will switch to TinyMCE, because for open-source it is limited to 1,000 editor loads per month and the CKEditor has many features behind a paywall, for example paste from Microsoft Word, which works in Quill v2.0.

**Reply by ThoSap** [reply]
See https://github.com/primefaces/primeng/issues/14721
